apps/wallet-mobile/src/WalletNavigator.tsx

```diff
@@ -129,31 +125,13 @@ const WalletTabNavigator = () => {
           )}
         </Tab.Screen>
 
-        <Tab.Screen
-          name="nfts"
-          options={{
-            tabBarIcon: ({focused}) => <Icon.Image size={28} color={focused ? colors.active : colors.inactive} />,
-            tabBarLabel: strings.nftsTabBarLabel,
-            tabBarTestID: 'nftsTabBarButton',
-          }}
-        >
-          {() => (
-            <SearchProvider>
-              <NftsNavigator />
```

Comment:
yes, you can check here -> https://www.figma.com/design/KeBVo6rWNYFLXwTphz9lHi/Mobile-App?node-id=24769-139696&m=dev

Reply:
It should be moved into the portfolio in any case, otherwise the nfts are inaccesible

Reply:
NFT are accesible from portfolio

Reply:
No they're not, because there's no navigator to reach them. The "links" in the portfolio sent them to the navigator stack you remove here, so now they don't work.

Reply:
ahhh good point. will review it

Reply:
If the Nfts now live inside navigator that's fine, but the contents of the stack need to be in the portfolio stack, and everything that linked there changed.

Reply:
fixed. thanks